MozillaFoundation/foundation.mozilla.org
# Include missing font weight and remove a google font reference

# Description

Two small font-weight regressions appeared on staging.

![Screenshot 2024-05-20 at 12 28 50 PM](https://github.com/MozillaFoundation/foundation.mozilla.org/assets/3083419/460d6ab4-2149-41d2-8198-29a46b71b9ad)
![Screenshot 2024-05-20 at 12 29 03 PM](https://github.com/MozillaFoundation/foundation.mozilla.org/assets/3083419/052795bc-6e05-442b-94ba-d0983c667d56)

This PR includes missing font weight and removes a stray google font reference.

Related PRs/issues: #12351

# To Test

1) Review the homepage(s). Ensure there's no font weight regression on the h2.
- https://foundation-s-self-hoste-sfhkvq.herokuapp.com
- https://foundation.mozilla.org/

2) Review the YouTube Regrets page(s). Ensure there's no font-weight regression on the body text.
- https://foundation-s-self-hoste-sfhkvq.herokuapp.com/en/campaigns/youtube-regrets/
- https://foundation.mozilla.org/en/youtube/regrets/


┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/TP1-686)
